cardano-diffusion/tests/lib/Test/Cardano/Network/Diffusion/Testnet.hs

```diff
@@ -2907,70 +2908,58 @@ prop_diffusion_target_active_below ioSimTrace traceNumber =
               events
 
           govActiveFailuresSig :: Signal (Set NtNAddr)
-          govActiveFailuresSig =
+          govActiveFailuresSig = holdUntilNextWakeup $
```

Comment:
Could you elaborate why `holdUntilNextWakeup` is the right thing to do with the `Signal.keyedLinger` here which holds recent async demotions?